scrape_data/cta_schedule_versions.py

```diff
@@ -0,0 +1,59 @@
+import boto3
+import sys
+import data_analysis.static_gtfs_analysis as sga
+import pendulum
+from io import StringIO
+
+ACCESS_KEY = sys.argv[1]
+SECRET_KEY = sys.argv[2]
+
+client = boto3.client(
+    's3',
+    aws_access_key_id=ACCESS_KEY,
+    aws_secret_access_key=SECRET_KEY
+)
+
+s3 = boto3.resource(
+    's3',
+    region_name='us-east-1',
+    aws_access_key_id=ACCESS_KEY,
+    aws_secret_access_key=SECRET_KEY
+)
+
+date = pendulum.now().to_date_string()
+
+zipfile, zipfile_bytes_io = sga.download_cta_zip()
+print(f'Saving zipfile available at '
+      f'https://www.transitchicago.com/downloads/sch_data/google_transit.zip '
+      f'on {date} to public bucket')
+zipfile_bytes_io.seek(0)
+client.upload_fileobj(zipfile_bytes_io, 'chn-ghost-buses-public', f'google_transit_{date}.zip')
+
+data = sga.download_extract_format()
+trip_summary = sga.make_trip_summary(data)
+
+route_daily_summary = (
+    sga.summarize_date_rt(trip_summary)
+)
+
+csv_buffer = StringIO()
+route_daily_summary.to_csv(csv_buffer)
+
+print(f'Saving cta_route_daily_summary_{date}.csv to public bucket')
+s3.Object('chn-ghost-buses-public', f'cta_route_daily_summary_{date}.csv')\
```

Comment:
Same comment as above regarding an intermediate directory level for this path, and this one is a bit trickier since we already generate these files from the current batched process (currently in `schedule_summaries/route_level`)... We probably will need to figure out how to do a cutover from the old process to new. 

Maybe `f'schedule_summaries/daily_job/`? 

Another question here would be whether we want to only save *that day's* activity (i.e, `route_daily_summary[route_daily_summary.date = {date}]`), because otherwise these files are pretty big to just save literally every day.

Reply:
Saving only that date would probably be good since we could concatenate the data later.